src/app/templates/app/license_namespace_information.html

```diff
@@ -0,0 +1,102 @@
+{% extends 'app/base.html' %}
+{% load static %}
+{% load staticfiles %}
+{% block body1 %}
+
+<div id="messages" class="messages">
+</div>
+<p class ="lead"> {{ medialink }}</p>
+<p class ="lead"> {{ error }}</p>
+<div class="panel panel-default">
+    {% if licenseInformation %}
+<div class="panel-heading"> <p class="lead">{{licenseInformation.fullname}}
+        {% if licenseInformation.archive %}
+            <span class="label label-success">Archived</span>
+        {% endif %}
+    </p>
+</div>
+    {% endif %}
+<div class="panel-body">
+    {% if licenseInformation %}
+    <div class="licenseField col-sm-12">
+        <p class="fieldName">Fullname</p>
+        <p>{{licenseInformation.fullname}}</p>
+    </div>
+    <div class="licenseField col-sm-12">
+        <p class="fieldName">License author name</p>
+        <p>{{licenseInformation.licenseAuthorName}}</p>
+    </div>
+    <div class="licenseField col-sm-12">
+        <p class="fieldName">Namespace</p>
+        <p>{{licenseInformation.namespace}}</p>
+    </div>
+    <div class="licenseField col-sm-12">
+        <p class="fieldName">Short Identifier</p>
+        <p>{{licenseInformation.shortIdentifier}}</p>
+    </div>
+    <div class="licenseField col-sm-12">
+        <p class="fieldName">Source/URL</p>
+        <p>{{licenseInformation.url}}</p>
+    </div>
+    <div class="licenseField col-sm-12">
+        <p class="fieldName">Publicly shared</p>
+        <p>{{licenseInformation.publiclyShared}}</p>
+    </div>
+    <div class="licenseField col-sm-12">
+        <p class="fieldName">Description</p>
+        <p>{{licenseInformation.description}}</p>
+    </div>
+    <div class="licenseField col-sm-12">
+        <p class="fieldName">Notes</p>
+        <p>{{licenseInformation.notes|safe}}</p>
+    </div>
+    <div class="licenseField col-sm-12">
+        <p class="fieldName">Submission Date</p>
+        <p>{{licenseInformation.submissionDatetime}}</p>
+    </div>
+    <div class="col-sm-12">
+        <div class="col-sm-12">
+            <a href="/app/license_requests/" class="btn">Go Back</a>
```

Comment:
The `href` URL should be of the namespaces page.

Reply:
@techytushar I have corrected this